src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -81,7 +82,16 @@ private string GetResultsFilePath(AssemblyInfo assemblyInfo)
                 // an empty log just in case, so our runner will still fail.
                 File.Create(resultsFilePath).Close();
 
-                var dumpOutputFilePath = Path.Combine(resultsDir, $"{assemblyInfo.DisplayName}.dmp");
+                // Define environment variables for processes started via ProcessRunner.
+                var environmentVariables = new Dictionary<string, string>();
+                environmentVariables.Add(ProcDumpRunner.ProcDumpPathEnvironmentVariableKey, _options.ProcDumpPath);
+
+                // Set environment variables for the current process.
```

Comment:
Removed this line. Thanks!

---
In reply to: [122764150](https://github.com/dotnet/roslyn/pull/20129#discussion_r122764150) [](ancestors = 122764150)